plenum/server/node.py

```diff
@@ -2055,6 +2075,16 @@ def process_query(self, request: Request, frm: str):
         result = handler.get_query_response(request)
         self.transmitToClient(Reply(result), frm)
 
+    def process_action(self, request, frm):
+        # Process a execute action request
+        try:
+            self.actionReqHandler.validate(request)
+            result = self.actionReqHandler.apply(request)
+            self.transmitToClient(Reply(result), frm)
+        except Exception as ex:
+            self.transmitToClient(Reject(request.identifier,
```

Comment:
Reject must be sent to the sender only in case dynamic validation (`validate`) has failed.